server/datastore/mysql/nanomdm_storage.go

```diff
@@ -0,0 +1,184 @@
+package mysql
```

Comment:
heads-up: 90% of this file was just moved from somewhere else. Only two new methods: `(s *NanoMDMStorage) EnqueueDeviceLockCommand` and a transactionable `func enqueueCommandDB`